src/compiler/crystal/interpreter/compiler.cr

```diff
@@ -2106,6 +2106,7 @@ class Crystal::Repl::Compiler < Crystal::Visitor
 
       {% if Debug::DECOMPILE %}
         puts "=== #{target_def.owner}##{target_def.name}#block ==="
+        puts compiled_block.local_vars
```

Comment:
Is this intentional or a leftover fragment?

Reply:
Ah, I forgot to mention this. It's intentional. It shows local vars a block. The same is done for defs, I just forgotten about it when doing blocks. I used it here to be able to debug this more easily.